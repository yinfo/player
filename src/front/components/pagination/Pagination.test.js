// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Pagination from './Pagination';

describe('Pagination component', () => {
  it('renders as expected', () => {
    const props = {
      size: 'large',
      numberOfPagination: 5,
    };
    const component = shallow(<Pagination {...props} />);
    expect(component).toMatchSnapshot();
  });
});
