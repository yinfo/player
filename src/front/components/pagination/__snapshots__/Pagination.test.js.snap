// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination component renders as expected 1`] = `ShallowWrapper {}`;
